electrolytes/preparesimulations.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash 
+
+num_lines=$(wc -l < elytes.csv)
+num_lines=$((num_lines-1))
+for ((i = 0; i < num_lines; i++)); do
+    mkdir $i
+    cp prepopenmmsim.py ./$i
+    cp lammps2omm.py ./$i
+    
+    #Generate and run short simulation of the solvent 
+    python generatesolvent.py $i
+    cd $i
+    python prepopenmmsim.py solvent
+    cp ../runsolvent.py ./
+    python runsolvent.py
+    cd ..
+    
```

Comment:
This shouldn't need to be repeated every time, right? There's only a finite number of solvent systems that we're looking at, so we should just be able to run those once, storing the densities and whatever other numbers we need, right?

Reply:
Yeah we talked about this, and you're right. I think it's just a matter of how we want to set up the workflow. Because at the end of the day, someone needs to run those first so I put that as part of the preparesimulations.sh